Dockerfile.stream8

```diff
@@ -16,4 +18,17 @@ RUN dnf install -y \
         make \
         rpm-build \
         tar
-COPY containerroot /
+
+COPY containerroot /workspace/
+
+# Download source rpms and install it's dependencies
+RUN dnf download --source \
+    ovirt-engine \
+    vdsm \
+    && \
+    dnf builddep -y *.src.rpm
+
+# Do a cleanup of old src rpm and make the container smaller
+RUN rm -f *.src.rpm \
```

Comment:
If you perform this step in a separate `RUN` it is not going to decrease the container size as it will be a separate layer.